python/ray/tune/trial.py

```diff
@@ -48,6 +50,53 @@ def __str__(self):
             return "{}:{}".format(self.hostname, self.pid)
 
 
+class TrialDirectory(object):
```

Comment:
Can you rename this? Or else I'm getting something confusing like:

trial.trial_dir.logdir
trial.logdir
trial.trial_dir


and it's not quite clear which is an object, which is a string, which is a path, etc.